about.md

```diff
@@ -6,294 +6,48 @@ title: About MDAnalysis
 <img src="{{ site.baseurl }}/public/mdanalysis-logo_square.png"
 style="float: right" alt="MDAnalysis" width="30%"/>
 
-## Mission
+MDAnalysis is an open-source Python library for analyzing molecular dynamics simulations and biomolecular structures. It enables high-performance, reproducible computational studies, supporting researchers across academia and industry.
 
-The MDAnalysis [Community](#community) is interested in all facets of **working
-with data in the computational molecular sciences**. We welcome everyone. **We all follow our
-[Code of Conduct] and strive to create an environment that is welcoming to
-all.** Our primary purpose is to produce software that scientists in academia and
-industry will trust to use in their research.
+Learn more about our mission on the [Mission]({{ site.baseurl }}/pages/mission/) page.
 
-We develop and maintain projects related to the broader goal of processing and
-analyzing data in the computational molecular sciences. We aim to empower users/developers to
-work with our packages following [FAIR principles]. Our central package is the
-[MDAnalysis library] for the analysis of computer simulations of many-body
-systems at the molecular scale. 
+## Get Involved
 
-We believe that scientific software should be open to all while using best
-practices to maintain high standards of correctness and reproducibility. We
-emphasize educating our users to make best use of the tools that we produce, to
-enable them to become contributors to our community and code bases.
+MDAnalysis is built and maintained by a global community of researchers, developers, and users. There are many ways to participate:
 
+- **Join the Community:** Connect with other users, report issues, and contribute through multiple channels. Learn more on our [Community]({{ site.baseurl }}/pages/community/) page.
+- **Contribute:** Whether through coding, documentation, or community support, get started with our [Contributing Guide]({{ site.baseurl }}/pages/contribute/).
+- **Follow Our Code of Conduct:** We foster a welcoming, inclusive environment. Read our [Code of Conduct]({{ site.baseurl }}/pages/conduct/).
 
+## Team & Leadership
 
-## Community
+MDAnalysis is maintained by [Core Developers][] who lead the project and make key decisions regarding its development. Past contributors who stepped back hold the title of Emeritus Core Developer.
```

Comment:
> the Core Developers

article

> Emeritus/Emerita

sounds a bit clunky but it would be incorrect to only use the male version